assets/js/wpuf-form-builder.js

```
@@ -834,6 +834,22 @@
 
                 e.clearSelection();
             });
+
+            // Mutual exclusivity between Enable Payments and Enable Pricing Fields Payment
+            var self = this;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'self'. Consider to delete it or rename it.

'self' is defined here:


(lint/suspicious/noRedeclare)